projects/components/src/multi-select/multi-select.component.ts

```diff
@@ -40,6 +40,13 @@ import { SelectSize } from '../select/select-size';
         </ht-popover-trigger>
         <ht-popover-content>
           <div class="multi-select-content">
+            <div class="multi-select-option all-options" (click)="this.onAllSelectionChange()">
```

Comment:
Can we make the presence of the all option controlled by an input?

Reply:
sure